src/polyfem/assembler/Assembler.hpp

```diff
@@ -36,11 +36,11 @@ namespace polyfem::assembler
 
 		virtual std::string name() const = 0;
 
-		int size() const { return size_; }
+		long int size() const { return size_; }
```

Comment:
there are probably ok, also changing the type and not where they are assigned probably dosent do anything